Apply-For-Me-Api/src/main/java/com/hydraulic/applyforme/controller/ProfessionalController.java

```diff
@@ -49,5 +48,11 @@ public Page<Professional> retrieveApplicants(@PathVariable int pageNo, @PathVari
         return  service.retrieveAllProfessionals(pageNo, pageSize);
 
     }
+
+    @GetMapping("/view-application-summary/{applicantEmail}")
+    public List<JobSummaryResponse> retrieveApplicantJobSummary(@PathVariable String applicantEmail){
+    return  service.retrieveProfessionalSubmissions(applicantEmail);
+    }
+
```

Comment:
Why are you using email of the user and not the user id as your path variable? also you do not need to pass the id as a path variable instead, fetch it from the security context.

Reply:
Done